src/app/debug.tsx

```diff
@@ -109,21 +113,55 @@ const Debug = (c: RenderContext) => {
                     <ul>
                         {images.map(([key, value]) => (
                             <li key={value.Id}>
-                                RepoTags: <strong>{value.RepoTags}</strong><br />Id: {value.Id}<br />Size: {humanSize(value.Size)}
+                                RepoTags: <strong>{shortHash(value.RepoTags.join(', '), 38)}</strong><br />Id: {shortHash(value.Id)}<br />Size: {humanSize(value.Size)}
                             </li>
                         ))}
                     </ul>
 
                     <figcaption>API</figcaption>
                 </figure>
+
+                <figure>
+                    <p>Containers</p>
+                    <ul>
+                        {(
+                            c.docker.containers
+                                .sort((a, b) => a.Names[0].localeCompare(b.Names[0]))
+                                .map(info => <li key={info.Id}><strong>{info.Names}</strong> - {shortHash(info.Id)}<br />Image ID: {shortHash(info.ImageID)}<br />Status: {info.Status}</li>)
+                        )}
+                    </ul>
+
+                    <p>Images</p>
+                    <ul>
+                        {(
+                            c.docker.images
+                                .sort((a, b) => a.RepoTags[0].localeCompare(b.RepoTags[0]))
+                                .map(info => <li key={info.Id}>RepoTags: <strong>{shortHash(info.RepoTags.join(', '), 38)}</strong><br />Id: {shortHash(info.Id)}</li>)
+                        )}
+                    </ul>
+                    <figcaption>Docker</figcaption>
+                </figure>
             </div>
         </Shell>
     );
 };
 
 type RenderContext = {
     startedServerAt: Date;
-};
-export default function renderDebug(renderContext: RenderContext) {
-    return ReactDOMServer.renderToStaticMarkup(<Debug {...renderContext} />);
+    docker: {
+        containers: Array<Dockerode.ContainerInfo>;
+        images: Array<Dockerode.ImageInfo>;
+    };
+}
+
+export default async function renderDebug({ startedServerAt }: { startedServerAt: Date }) {
+    return ReactDOMServer.renderToStaticMarkup((
+        <Debug
+            startedServerAt={startedServerAt}
+            docker={{
+                containers: await Docker.listContainers(),
```

Comment:
Do we want to list _all_ containers? Or just running ones?

Reply:
oh oops - good catch - I'll update it